### DISCUSSION TITLE
Why there is no EAN, MPN, Brand, Model name areas in core by default

### DISCUSSION THREAD

**Discussion by betayazilim** [woocommerce/woocommerce#49587] [General]
Hi there,
Google is very important for all of us. Google's merchant requires model name, brand, ean and MPN to list products.
Woocommerce is most popular platform but doesn't support these attributes.
All of merchant's try to find different solutions and there is generic solution.
If this is by purpose to make some money, which is fine.
It can be 1 plugin like woocommerce brands but that plugin should include EAN, MPN, Model name like areas too.

**Comment by barryhughes** [comment]
Hi! Please see [this earlier discussion](https://github.com/woocommerce/woocommerce/discussions/47519), which is relevant. In short, though, yes this is something we are considering.

**Reply by betayazilim** [reply]
Thanks a lot.
